jalali_date/admin.py

```diff
@@ -25,6 +28,42 @@ def __init__(self, *args, **kwargs):
         self.formfield_overrides = formfield_overrides
         super(ModelAdminJalaliMixin, self).__init__(*args, **kwargs)
 
+    def get_list_display(self, request):
+        if not settings.JALALI_DATE_DEFAULTS['LIST_DISPLAY_AUTO_CONVERT']:
```

Comment:
`LIST_DISPLAY_AUTO_CONVERT` setting has been added  in `JALALI_DATE_DEFAULTS` in `jalali_date\settings.py`. so if users override `JALALI_DATE_DEFAULTS` and don't set `LIST_DISPLAY_AUTO_CONVERT`, it will be [added automatically](https://github.com/a-roomana/django-jalali-date/blob/c227713a1fe4cf2a6421abe99dc0a662214e5e31/jalali_date/settings.py#L34).